static/app/icons/index.tsx

```diff
@@ -94,6 +94,8 @@ export {IconResize} from './iconResize';
 export {IconRewind10} from './iconRewind10';
 export {IconSad} from './iconSad';
 export {IconSearch} from './iconSearch';
+export {IconSeer} from './iconSeer';
+export {IconSeerLoading, IconSeerWaiting} from './iconSeerLoading';
```

Comment:
@roaga can we make these icons something like `<IconSeer state="loading|waiting"/>`, or are they so different that they warrant separate icons?

Reply:
i'll merge all three and add the `variant: undefined | 'loading' | 'waiting'` prop

Reply:
@ryan953 I'm going to approve this and we can tackle it in a separate PR btw, so it's easier to review and we can keep this PR only about moving seer to the icon set